bgpd/bgp_mplsvpn.c

```diff
@@ -997,10 +1036,18 @@ static void vpn_leak_to_vrf_update_onevrf(struct bgp *bgp_vrf,       /* to */
 			   num_labels);
 	}
 
+	/* For VRF route-leaking, the source will be the originating VRF. */
```

Comment:
Should this comment read "For VRF-to-VRF route-leaking..." ?

Reply:
fixed